src/gps_to_pose_conversion_node.cpp

```diff
@@ -184,6 +201,8 @@ int main(int argc, char **argv) {
                     g_covariance_orientation_z, 0.11);
   ros::param::param<std::string>("~frame_id",
                                  g_frame_id, "world");
+  ros::param::param<std::string>("~tf_child_frame_id",
```

Comment:
I kept on using `ros::param::param` for consistency with previous parameters loading. I didn't want to change that for only this parameter.